sdk/go/common/resource/plugin/provider.go

```diff
@@ -52,8 +52,8 @@ type (
 	}
 )
 
-func (ParameterizeArgs) isParameterizeParameters()  {}
-func (ParameterizeValue) isParameterizeParameters() {}
+func (*ParameterizeArgs) isParameterizeParameters()  {}
```

Comment:
This is so you don't need to cover ParameterizeArgs and *ParameterizeArgs in the type switch statements. They've _got_ to be pointers.

Reply:
Stupid questions -- what are these functions for? They don't have a return value or an implementation, and you can distinguish using a type switch? If they do have a use, worth documenting it here?

Reply:
isParameterizeParameters is a private method so nothing outside the package can implement it, so we can know the only types that implement ParameterizeParameters are *ParameterizeArgs and *ParameterizeValue.

Reply:
Ahhhh neat. Thanks!